dataclass_wizard/utils/typing_compat.py

```diff
@@ -372,8 +364,12 @@ def eval_forward_ref(base_type: FREF,
     # Evaluate the ForwardRef here
     base_globals = sys.modules[cls.__module__].__dict__
 
-    # noinspection PyProtectedMember
-    return typing._eval_type(base_type, base_globals, _TYPING_LOCALS)
+    if sys.version_info >= (3, 13):
+        # noinspection PyProtectedMember
+        return typing._eval_type(base_type, base_globals, _TYPING_LOCALS, type_params=())
```

Comment:
@benjjs Minor thing, can you pull this out to the top level? That way we won't need to do a python version check each time `eval_forward_ref` is called. Thanks!

```
if sys.version_info >= (3, 13):
  _eval_type = functools.partial(typing._eval_type, type_params=())
else:
  _eval_type = typing._eval_type
```

Reply:
Follow-up can we please add this to `utils/typing_compat.py` and import/use it instead:

```
# Check if currently running Python 3.8
PY313 = _PY_VERSION == (3, 13)
```

Thanks!

Reply:
> @benjjs Minor thing, can you pull this out to the top level? That way we won't need to do a python version check each time `eval_forward_ref()` is called. Thanks!
> 
> ```
> if sys.version_info >= (3, 13):
>   _eval_type = functools.partial(typing._eval_type, type_params=())
> else:
>   _eval_type = typing._eval_type
> ```

This was not resolved, but its OK, I can do that. Simple fix.